modules/velero-obtaining-by-downloading.adoc

```diff
@@ -23,16 +23,17 @@ The page includes instructions for:
 
 . Open a browser and navigate to link:https://{velero-domain}/docs/v{velero-version}/basic-install/#install-the-cli["Install the CLI" on the Verleo website].
 . Follow the appropriate procedure for macOS, GitHub, or Windows.
-. Download the Velero version appropriate for your version of OADP, according to the table that follows:
+. Download the Velero version appropriate for your version of OADP and {product-title}  according to the table that follows:
 +
-.OADP-Velero version relationship
-[cols="2", options="header"]
+.OADP-Velero-{product-title} version relationship
+[cols="3", options="header"]
 |===
-|OADP version |Velero version
-|0.2.6 |1.6.0
-|0.5.5 |1.7.1
-|1.0.0 |1.7.1
-|1.0.1 |1.7.1
-|1.0.2 |1.7.1
-|1.0.3 |1.7.1
+|OADP version |Velero version |{product-title} version
+|1.0.0 |link:https://1.7/docs/v1.7/[1.7] |4.6 and later
+|1.0.1 |link:https://1.7/docs/v1.7/[1.7] |4.6 and later
+|1.0.2 |link:https://1.7/docs/v1.7/[1.7] |4.6 and later
+|1.0.3 |link:https://1.7/docs/v1.7/[1.7] |4.6 and later
```

Comment:
The URLs here are invalid. Guessing maybe missing some attributes like the others below are using?

Reply:
@adellape Fixed -- thanks!